### PR TITLE
Upgarde viirs_aod2ioda.py.in with bias/error calculation, maskout and…

### PR DESCRIPTION
**Pull request by YoulongXia-NOAA**
Upgarde viirs_aod2ioda.py.in with nesdis bias/error calculation (default=none), maskout (default=none) and thinning (value is within 0.0 to 1.0, default=none) option, and new input/output netcdf files with high quality data and qcpath flag, and the changes of corresponding README.md and CMakeLists.txt.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,6 @@ Usage: odbapi2json.py [-h] [-c] [-q] input_odbapi output_temp > output.json
 The chem converters include all converter scripts for arosols and related chemistry variables. Currently only one Python script, `viirs_aod2ioda.py`, is used to convert the native netCDF format for observations of optical depth from VIIRS AOD550 to IODA netCDF format. Note that it takes only AOD550 explicitly and does not take the 11 AOD channels from VIIRS. The converter uses the following format to execute:
```
Comment (CoryMartin-NOAA) [inline README.md:134]: this was missed in the past review, change "arosols" to "aerosols"

File: README.md [hunk 2]
```diff
@@ -134,6 +134,6 @@ Usage: odbapi2json.py [-h] [-c] [-q] input_odbapi output_temp > output.json
...
  
 ```
-Usage: <converter.py> -i INPUT_FILE(S) -o OUTPUT_FILE
+Usage: <converter.py> -i INPUT_FILE(S) -o OUTPUT_FILE -m nesdis -k maskout -t 0.0
```
Comment (CoryMartin-NOAA) [inline README.md:137]: Add some more detail here to explain what the additional options mean and the defaults.

**Comment by YoulongXia-NOAA** [comment]
Hi Cory,

Add more explanations as you requested. I pushed it to my
features/viirs-aod. Thanks,

Youlong
================================================================
Youlong Xia, Ph.D
I. M. Systems Group at EMC/NCEP
5830 University Research Court
College Park, MD 20740
Phone: 301-683-3696
=================================================================

File: README.md [hunk 3]
```diff
@@ -131,9 +131,9 @@ Usage: odbapi2json.py [-h] [-c] [-q] input_odbapi output_temp > output.json
...
+The chem converters include all converter scripts for aerosols and related chemistry variables. Currently only one Python script, `viirs_aod2ioda.py`, is used to convert the native netCDF format for observations of optical depth from VIIRS AOD550 to IODA netCDF format. Note that it takes only AOD550 explicitly and does not take the 11 AOD channels from VIIRS. The converter uses the following format to execute:
  
 ```
 Usage: <converter.py> -i INPUT_FILE(S) -o OUTPUT_FILE -m nesdis -k maskout -t 0.0
 ```
-
+For method option (-m) of bias and uncertainty calculation (default/nesdis), deafult means to set bias and uncertainty as 0.0 and nesdis meansto use NESDIS bias and uncertainty calculation method. For maskout option (-k) default/maskout, default means to keep all missing values and maskout means to mask all missing values out. For thinning option, it value is within 0.0 and 1.0 depending how much data will be thinned, and 0.0 means without any thining.
```
Comment (CoryMartin-NOAA) [inline README.md:139]: "For thinning option, it value is " should probably say "For thinning option, the value should be " within 0.0 and 1.0...

**Comment by CoryMartin-NOAA** [comment]
@mpagowski you need a "good" python installation.
Try:
module use -a /scratch2/NCEPDEV/marineda/Jong.Kim/save/modulefiles/
module load anaconda/3.15.1

**Comment by YoulongXia-NOAA** [comment]
I used the following modules below:

module purge
module use -a /scratch1/NCEPDEV/jcsda/Ryan.Honeyager/jedi/modules
module load jedi-stack/intel-impi-18.0.5
module use -a /scratch2/NCEPDEV/marineda/Jong.Kim/save/modulefiles
module load anaconda/3.15.1.1

Thank you.

Youlong
================================================================
Youlong Xia, Ph.D
I. M. Systems Group at EMC/NCEP
5830 University Research Court
College Park, MD 20740
Phone: 301-683-3696
=================================================================

**Pull request merged**